src/renderer/prefComponents/image/index.vue

```diff
@@ -15,10 +15,15 @@
     </section>
     <separator></separator>
     <section class="image-folder">
-      <div class="description">Local image folder</div>
-      <div class="path">{{imageFolderPath}}</div>
+      <text-box
+        description="Local image folder"
+        :input="imageFolderPath"
+        :regexValidator="/^(?:$|([a-zA-Z]:)?[\/\\].*$)/"
+        :defaultValue="FolderPathPlaceholder"
+        :onChange="value => modifyImageFolderPath(value)"
+      ></text-box>
       <div>
-        <el-button size="mini" @click="modifyImageFolderPath">Modify</el-button>
+        <el-button size="mini" @click="modifyImageFolderPath()">Modify</el-button>
```

Comment:
You need to pass the `imageFolderPath` here, otherwise the path is `null`.

Reply:
This is intentional, path being null tells `editor.vue@imageAction` to open a dialog.